README.md

```diff
@@ -1,2 +1,79 @@
-# BetterFleet
\ No newline at end of file
+![image](/frontend/src/assets/banners/banner.png)
+
+# BetterFleet
+
+----
+
+## About BetterFleet
+
+BetterFleet is a free open source application designed to enhance the gaming experience in Sea Of Thieves by
+facilitating the creation of alliances among players. With BetterFleet, you can organize game sessions and invite your
+friends to join in a simple and intuitive manner
+
+**Important Note:** BetterFleet is not an official application of Sea Of Thieves. It was developed by the community for
+players looking to improve their gaming experience.
+--- 
+
+## Features
+
+- **Session Management:** The application allows users to create sessions capable of handling multiple players. It offers functionality to manage players and enhance the probability of finding a server. Additionally, it includes a synchronization mechanism that aligns all player clicks to increase the chances of landing on the same server.
+
+- **Self-Hosted Backend:** Users have the possibility to host the backend themselves, thanks to the app being open source. This feature ensures greater control over the application's deployment and maintenance.
+
+- **Statistics Tracking:** The application provides users with the ability to track statistics, offering insights into the potential percentage chance of finding a server. This feature helps users to assess their success rate and make informed decisions.
+
+---
+
+## Comparison of Fleet Management Applications: BetterFleet vs. FleetCreator
+
+| Feature                               | BetterFleet                    | FleetCreator                   |
+|---------------------------------------|--------------------------------|--------------------------------|
+| IPv6 support                          | :white_check_mark:             | :x:                            |
+| Less performance usage?               | :white_check_mark:             | :x:                            |
+| No ads                                | :white_check_mark:             | :x:                            |
+| No paid tiers                         | :white_check_mark:             | :x:                            |
+| UX friendly                           | :white_check_mark:             | :x:                            |
+| Open source = safer = transparent     | :white_check_mark:             | :x:                            |
+| Automatic click between the same crew | :white_check_mark:             | :white_check_mark:             |
+| No memory leak                        | :white_check_mark:             | :warning: (See note below)     |
+| Faster?                               | :question: (To be benchmarked) | :question: (To be benchmarked) |
+| Size of file?                         | :question: (To determine)      | :question: (To determine)      |
+
+**Note:** The :warning: symbol indicates there was a reported issue with FleetCreator using 8GB of RAM after 10 hours of
```

Comment:
It makes really like journalist and is weirdly said.. "Someone told us that.."
I think an image is worth 10 thousands words. Just put the screenshot of the RAM usage in a spoiler